### ISSUE TITLE
Merging requests in the Web UI needs only Maintenance ID

### ISSUE THREAD

**Issue by celan** [openSUSE/open-build-service#1065]
When trying to merge a maintenance request with an existing Maintenance incident it fails because it expects only the ID. In this case it needed '1035' and it rejected 'SUSE:Maintenance:1035'. The search field proposed the latter string. It doesn't find anything when providing only the ID.

**Comment by adrianschroeter** [comment]
which interface are you using here? The webui?

**Comment by celan** [comment]
Yes, that is in the Web UI.

**Comment by bgeuken** [comment]
SUSE:Maintenance:1035 is a project. The ui expects a request id of a request you want to merge it with.

**Comment by bbrunner** [comment]
@bgeuken: No, it expects the ID of the maintenance-project. It's about "Merge with existing incidents" in requests with the type incident.

**Comment by bgeuken** [comment]
Ah, you are right the [id parameter](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/request_controller.rb#L439) I saw comes from a hidden field tag:/

**Comment by bgeuken** [comment]
I think it's caused by [this line](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/bs_request.rb#L728). But the code is so complicated that I am really unsure about this. And there are no tests that would allow me to check this easily.
